packages/launch-config/package.json

```
@@ -0,0 +1,47 @@
+{
+    "name": "@sap-ux/launch-config",
+    "version": "1.0.0",
```

@broksy, could we start with a 0.x version in case we need changes?
```suggestion
    "version": "0.0.1",
```